tests/syntax/test_syntax.py

```diff
@@ -15,13 +15,15 @@
 
 @pytest.mark.parametrize( "f", files)
 def test_syntax(f):
+    # reset Errors singleton
+    errors = Errors()
+    errors.reset()
 
     pyccel = Parser(f)
     pyccel.parse()
 
-    # reset Errors singleton
-    errors = Errors()
-    errors.reset()
+    # Assert syntactic success
+    assert(not errors.has_errors())
```

Comment:
Can we please remove these unnecessary parentheses?

Reply:
Fixed in 3a24360